about.html

```diff
@@ -0,0 +1,126 @@
+                    <li class="contact-li"><a href="">sign Up</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="header-container">
+            <div class="header-container">
+                <div class="nav-container">
+                    <div class="logo">
+                        <div>
+                            <ul>
+                                <li>
+                                    <i class="fa-solid fa-graduation-cap cap"></i>
+                                    <a class="logo-link" href="home.html">Educational Resources</a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+
+                    <div class="nav">
+                        <ul class="nav-ul">
+                            <li class="navigation"><a href="home.html">Home</a></li>
+                            <li class="navigation"><a href="about.html">About</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="mobile-menu">
+                        <ul class="menu-list">
+                            <li><a class="mobile-link" href="home.html">Home</a></li>
+                            <li><a class="mobile-link" href="about.html">About</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="hamburger">
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section>
+        <div class="headline-container">
+            <p class="headline-p">"Hello! scholar"</p>
+            <h1>Educational Resources Summit 2023</h1>
+            <p class="headline-summary">
+                Educational resources are used in a learning environment to help and assist with people’s development
+                and
+                learning. We are designed to reinforce learning and in some cases allow people to put their knowledge to
+                the
+                test. Educational resources are brilliant for teachers and educators to help them deliver the best
+                quality
+                lessons.
+            </p>
+            <P class="venue">
+                Please contacts us via Email for any further questions about educational resources 2023!
+                <a href="" class="link">educationalresources@edu.ng</a>
+            </P>
+        </div>
+        <div class="theme">
+            <h2>Theme</h2>
+            <p class="theme-p">
+                "Success occurs when opportunity meets preparation".
+            </p>
+            <p class="summit-logo">
+                <i class="fa-solid fa-graduation-cap cap"></i>
+                <a class="logo-link" href="">Educational Resources</a>
+            </p>
+        </div>
```

Comment:
- [ ] I think it would be better if you wrap the selected code above in a new `section` tag for good semantics reasons as it is different from the hero section and make sure you stick its design to the one from the template. You can check the spacing between && around different elements, fonts, sizes, heights on different text,...